docs/guides/integration/docker.md

```diff
@@ -18,90 +18,53 @@ $ docker run ghcr.io/astral-sh/uv --help
 
 ### Available images
 
-uv builds and publishes the following Docker tags:
+uv provides a distroless Docker image including the `uv` binary. The following tags are published:
 
 - `uv:latest`
 - `uv:{major}.{minor}.{patch}`, e.g., `uv:0.4.4`
 - `uv:{major}.{minor}`, e.g., `uv:0.4` (the latest patch version)
 
-In addition, uv builds and publishes these additional tags:
+In addition, uv publishes the following images:
 
+<!-- prettier-ignore -->
 - Based on `alpine:3.20`:
-  - `uv:alpine`
-  - `uv:alpine3.20`
-  - `uv:{major}.{minor}-alpine`
-  - `uv:{major}.{minor}-alpine3.20`
-  - `uv:{major}.{minor}.{patch}-alpine`
-  - `uv:{major}.{minor}.{patch}-alpine3.20`
+    - `uv:alpine`
+    - `uv:alpine3.20`
```

Comment:
cc @samypr100 need to ignore prettier to get the four space indents that are required for proper subitem rendering 😭